GS01SUMO/GS01SUMO_R00004_ingest.csv

```diff
@@ -0,0 +1,80 @@
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/instruments/dcl11/imm/CTDMO_sn37-13557/*.hex,GS01SUMO-RII11-02-CTDMOQ015,recovered_inst,Available,
+mi.dataset.driver.ctdmo_ghqr.imodem.ctdmo_ghqr_imodem_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/cg_data/dcl11/imm/ctdmo06/*.DAT,GS01SUMO-RII11-02-CTDMOQ016,recovered_host,Available,
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/instruments/dcl11/imm/CTDMO_sn37-13577/*.hex,GS01SUMO-RII11-02-CTDMOQ016,recovered_inst,Available,
+mi.dataset.driver.ctdmo_ghqr.imodem.ctdmo_ghqr_imodem_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/cg_data/dcl11/imm/ctdmo07/*.DAT,GS01SUMO-RII11-02-CTDMOQ017,recovered_host,Available,
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/instruments/dcl11/imm/CTDMO_sn37-13588/*.hex,GS01SUMO-RII11-02-CTDMOQ017,recovered_inst,Available,
+#mi.dataset.driver.ctdmo_ghqr.imodem.ctdmo_ghqr_imodem_recovered_driver,,GS01SUMO-RII11-02-CTDMOQ031,recovered_host,Not Deployed,Not deployed
+#mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,,GS01SUMO-RII11-02-CTDMOQ031,recovered_inst,Not Deployed,Not deployed
+#mi.dataset.driver.ctdmo_ghqr.imodem.ctdmo_ghqr_imodem_recovered_driver,,GS01SUMO-RII11-02-CTDMOQ033,recovered_host,Not Deployed,Not deployed
+#mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,,GS01SUMO-RII11-02-CTDMOQ033,recovered_inst,Not Deployed,Not deployed
+mi.dataset.driver.ctdmo_ghqr.imodem.ctdmo_ghqr_imodem_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/cg_data/dcl11/imm/ctdmo08/*.DAT,GS01SUMO-RII11-02-CTDMOR018,recovered_host,Available,
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/instruments/dcl11/imm/CTDMO_sn37-12565/*.hex,GS01SUMO-RII11-02-CTDMOR018,recovered_inst,Available,
+mi.dataset.driver.ctdmo_ghqr.imodem.ctdmo_ghqr_imodem_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/cg_data/dcl11/imm/ctdmo09/*.DAT,GS01SUMO-RII11-02-CTDMOR019,recovered_host,Available,
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/instruments/dcl11/imm/CTDMO_sn37-12567/*.hex,GS01SUMO-RII11-02-CTDMOR019,recovered_inst,Available,
+mi.dataset.driver.ctdmo_ghqr.imodem.ctdmo_ghqr_imodem_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/cg_data/dcl11/imm/ctdmo10/*.DAT,GS01SUMO-RII11-02-CTDMOR020,recovered_host,Available,
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/instruments/dcl11/imm/CTDMO_sn37-13648/*.hex,GS01SUMO-RII11-02-CTDMOR020,recovered_inst,Available,
+mi.dataset.driver.dosta_abcdjm.ctdbp_p.dcl.dosta_abcdjm_ctdbp_p_dcl_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/cg_data/dcl11/imm/ctdbp01/ctdbp01*.DAT,GS01SUMO-RII11-02-DOSTAD031,recovered_host,Available,

```

Comment:
For the DOSTA and FLORD attached to the CTDBP, we only have a recovered host data set? Is there not a driver for a recovered instrument dataset?

Reply:
That has been the case so far. Looks like there are not any recovered-inst streams in the system for the DOSTA (GS01SUMO-RII11-02-DOSTAD03X) or FLORD (GS01SUMO-RII11-02-FLORDG03X) that are attached to the CTDBP. 

I thought that both DOSTA and FLORD do not record data internally so then it would make sense that they would not have any recovered instrument data? Or are those data somehow logged in the CTDBP hex file?

Reply:
They should be logged in the CTDBP hex file.  So if inductive comms crap out, the only source of the DOSTA and FLORD data is in the CTDBP.  This may need a Redmine ticket.